src/airtableApi.jsx

```diff
@@ -0,0 +1,77 @@
+import {
+  AIRTABLE_TABLE_NAME,
+  AIRTABLE_BASE_ID,
+  AIRTABLE_API_KEY,
+} from './configuration.jsx';
+
+const tableUrl = `https://api.airtable.com/v0/${AIRTABLE_BASE_ID}/${AIRTABLE_TABLE_NAME}`;
+
+const defaultHeaders = {
+  Authorization: `Bearer ${AIRTABLE_API_KEY}`,
+};
+export const listRecords = async options => {
```

Comment:
The async await syntax is a different way of handling promises. Here is comparable snippet of using the promise callback handlers that operates similarly.

```javascript
export const listBlogs = () => {
  return fetch(
    `https://api.airtable.com/v0/${AIRTABLE_BASE_ID}/${AIRTABLE_TABLE_NAME}`,
    {
      headers: {
        Authorization: `Bearer ${AIRTABLE_API_KEY}`,
      },
    }
  )
    .then(response => {
      return response;
    })
    .catch(error => {
      console.log(error);
      // handle the error here
    });
};
```